third_party/terraform/data_sources/data_source_google_compute_resource_policy.go

```diff
@@ -0,0 +1,62 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleComputeResourcePolicy() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleComputeResourcePolicyRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"region": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"self_link": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceGoogleComputeResourcePolicyRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+	region, err := getRegion(d, config)
+	if err != nil {
+		return err
+	}
+	name := d.Get("name").(string)
+	resourcePolicy, err := config.clientCompute.ResourcePolicies.Get(project, region, name).Do()
```

Comment:
s/clientCompute/clientComputeBeta/g

Reply:
Hey,

thank you for having a look at my PR! The use of `clientCompute` in the datasource is actually on purpose. When the test run creates a `google_compute_resource_policy` resource via Terraform, Terraform seems to use the stable API already to create this resource. 

If I use `clientComputeBeta` for accessing the beta API, this error occurs when running the test: 
```
➜  terraform-provider-google-beta git:(master) ✗ make testacc TEST=./google-beta TESTARGS='-run=TestAccDataSourceComputeResourcePolicy'
==> Checking source code against gofmt...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google-beta -v -run=TestAccDataSourceComputeResourcePolicy -timeout 240m -ldflags="-X=github.com/terraform-providers/terraform-provider-google-beta/version.ProviderVersion=acc"
=== RUN   TestAccDataSourceComputeResourcePolicy
=== PAUSE TestAccDataSourceComputeResourcePolicy
=== CONT  TestAccDataSourceComputeResourcePolicy
--- FAIL: TestAccDataSourceComputeResourcePolicy (10.57s)
    testing.go:568: Step 0 error: Check failed: Check 1/1 error: self_link is https://www.googleapis.com/compute/beta/projects/<my-project>/regions/us-central1/resourcePolicies/policy; want https://www.googleapis.com/compute/v1/projects/<my-project>/regions/us-central1/resourcePolicies/policy
FAIL
FAIL	github.com/terraform-providers/terraform-provider-google-beta/google-beta	10.588s
make: *** [GNUmakefile:17: testacc] Error 1
```

For the test to work, terraform-providers/terraform-provider-google-beta/pull/959 needs to be merged into the `terraform-provider-google-beta` repo first. It updates the google API dependency to the latest version. If you want to double check this, you can run the test against [my fork](https://github.com/JanMa/terraform-provider-google-beta).

Kind regards,
Jan